cockroachdb/example/cert-manager/db-init-manifests/example-database/init.yaml

```diff
@@ -14,3 +14,11 @@ spec:
             # name of the user that the init job will grant access on the DB to
             - name: DB_USER
               value: "example_user"
+---
+apiVersion: cert-manager.io/v1
```

Comment:
is this strictly part of the `init` process, or more general? I'm wondering if it would be clearer if this were in its own file like `cert.yaml`

Reply:
done